src/frontend/efiling-frontend/src/components/page/home/Home.js

```diff
@@ -81,13 +75,25 @@ const checkCSOAccountStatus = (
         ).identifier;
         sessionStorage.setItem("csoAccountId", csoAccountIdentifier);
         setCsoAccountStatus({ isNew: false, exists: true });
+      } else {
+        axios
+          .get("/bceidAccount")
```

Comment:
Do we not need to pass a submission id or anything to this endpoint?

Reply:
nope!